locales/id/translation.json

```
@@ -0,0 +1,270 @@
+{
+    "$Language": {
+        "en": "English",
+        "pt-BR": "Português - Brasil",
+        "es": "Español",
+        "it": "Italiano",
+        "zh-TW": "繁體中文",
+        "de-DE": "Deutsch",
+        "hi": "हिंदी",
+        "mr": "मराठी",
+        "id": "Bahasa Indonesia"
+    },
+    "$Preferences": {
+        "title": "Time to Leave - Pengaturan",
+        "userPreferences": "Pengaturan Akun",
+        "workingDays": "Hari Kerja",
+        "sun": "Min",
+        "mon": "Sen",
+        "tue": "Sel",
+        "wed": "Rab",
+        "thu": "Kam",
+        "fri": "Jum",
+        "sat": "Sab",
+        "hideNonWorkingDay": "Sembunyikan hari Bukan-kerja (Tampilan Bulan)",
+        "hoursPerDay": "Jam per-hari",
+        "appBehavior": "Perilaku Aplikasi",
+        "notification": "Notifikasi",
+        "allowRecurringNotifications": "Izinkan pemberitahuan berulang",
+        "minutesBetweenNotifications": "Menit antar pemberitahuan",
+        "startOnLogin": "Mulai saat Login",
+        "countTodayOnTotals": "Total Hitungan hari",
+        "overallBalanceStart": "Rata Rata Keseluruhan Tanggal Mulai ",
+        "minimizeToTray": "Tombol Minimalkan harus meminimalkan tray",
+        "closeMinimizeToTray": "Tombol Tutup harus diminimalkan ke tray",
+        "themes": "Tema",
+        "systemDefault": "Sistem Default",
+        "light": "Terang",
+        "dark": "Gelap",
+        "cadentStar": "Cadent Star",
+        "view": "Tampilan",
+        "month": "Bulan",
+        "day": "Hari",
+        "numberOfEntries": "Jumlah Entri",
+        "fixed": "Tetap",
+        "flexible": "Fleksibel",
+        "language": "Bahasa",
+        "hours-per-day": "HH:mm"
+    },
+    "$Menu": {
+        "about": "Tentang",
+        "all-clear": "Semua Bersih!",
+        "all-files": "Semua File",
+        "cancel": "Batal",
+        "check-for-updates": "Periksa Update",
+        "clear-database": "Bersihkan Database",
+        "confirm-clear-all-data": "Anda yakin Menghapus semua data?",
+        "confirm-import-db": "Anda yakin ingin mengimpor database? ini akan menimpa informasi terkini.",
+        "confirm-migrate-to-flexible": "Anda yakin ingin memigrasi basis data kalender tetap ke kalender fleksibel? \n\nBasis data kalender fleksibel yang ada akan dihapus.",
+        "copy": "Salin",
+        "could-not-be-loaded": "Tidak dapat dimuat",
+        "cut": "Pindah",
+        "database-export": "Ekspor Database",
+        "database-imported": "Impor Database ,
```

You are missing a " here:

```suggestion
        "database-imported": "Impor Database",
```